src/openrct2/world/Map.cpp

```diff
@@ -1540,6 +1540,9 @@ void map_restore_provisional_elements()
 void map_remove_out_of_range_elements()
 {
     int32_t mapMaxXY = gMapSizeMaxXY;
+    //#9955 ensure that we can remove elements
+    bool buildState = gCheatsBuildInPauseMode;
+    CheatsSet(CheatType::BuildInPauseMode, true);
 
     for (int32_t y = 0; y < (MAXIMUM_MAP_SIZE_TECHNICAL * 32); y += 32)
```

Comment:
This can still go wrong if the user does not have cheats privileges. Perhaps it should check `gCheatsBuildInPauseMode` after CheatsSet, moving the for loop into an if.

Reply:
There's also an alternative where we can just unpause the game for that tick.  Are there pause / unpause permissions for multiplayer as well?  The reason I didn't implement the fix by unpausing was to reduce any side effects on the game.  

For now I'll implement your suggestion and see if that fixes it.